charts/prometheus-ipmi-exporter/values.yaml

```
@@ -63,6 +63,11 @@ serviceMonitor:
 
   # Configuration file for ipmi_exporter
 
+useSecret: false
+secret:
+  name: ipmi-secret
+  key: config.yml
```

We should be a bit more specific of the secret. If using a map for a single feature, I'd suggest putting the toggle in the map.

```suggestion
configSecret:
  enabled: false
  name: ipmi-secret
  key: config.yml
```